.gitignore

```
@@ -3,8 +3,9 @@ node_modules
 
 .vscode/*
 .history
+.DS_Store
```

```suggestion
**/.DS_Store
```
This ignores all `.DS_Store` files in all folders